libs/utils/analysis/latency_analysis.py

```diff
@@ -49,6 +51,26 @@ def __init__(self, trace):
 
     @memoized
     def _dfg_latency_df(self, task):
+        """
+        DataFrame of task's wakeup/suspend events
+
+        The returned DataFrame has these columns
+        - Time: the time an event related to this task happened
```

Comment:
This isn't actually a column in the DataFrame it's just the index, I don't think you need to mention it here.